runtime-coreclr outerloop pipeline failing osx-arm64 job

It appears the osx-arm64 job was enabled by https://github.com/dotnet/runtime/pull/43187

The "CoreCLR Pri1 Runtime Tests Run OSX arm64 checked" job has been failing since then with:
```
/Users/runner/work/1/s/eng/common/msbuild.sh --ci /Users/runner/work/1/s/src/tests/Common/helixpublishwitharcade.proj /maxcpucount /bl:/Users/runner/work/1/s/artifacts/log/SendToHelix.binlog /p:TargetArchitecture=arm64 /p:TargetOS=OSX /p:TargetOSSubgroup= /p:Configuration=Checked
========================== Starting Command Output ===========================
/bin/bash --noprofile --norc /Users/runner/work/_temp/f9dc95ff-20f7-492b-9d42-f5683b6c9d1c.sh
/Users/runner/work/1/s/.dotnet/sdk/5.0.100-rc.1.20454.5/MSBuild.dll /nologo -logger:/Users/runner/work/1/s/.packages/microsoft.dotnet.arcade.sdk/5.0.0-beta.20506.7/tools/netcoreapp2.1/Microsoft.DotNet.Arcade.Sdk.dll -maxcpucount /m /maxcpucount -verbosity:m /v:minimal /bl:/Users/runner/work/1/s/artifacts/log/SendToHelix.binlog /clp:Summary /nr:false /p:TreatWarningsAsErrors=true /p:ContinuousIntegrationBuild=true /p:TargetArchitecture=arm64 /p:TargetOS=OSX /p:TargetOSSubgroup= /p:Configuration=Checked /warnaserror /Users/runner/work/1/s/src/tests/Common/helixpublishwitharcade.proj
  DotNetCliVersion: 5.0.0-rc.1.20451.14
  DotNetCliPackageType: runtime
  HelixRuntimeRid: osx-arm64
  Determining projects to restore...
  Restored /Users/runner/work/1/s/src/tests/Common/xunitconsolerunner.depproj (in 1.69 sec).
  a eventprovider
  a eventprovider/eventprovidertest
  a exception_handling
  a exception_handling/pal_sxs
  a exception_handling/pal_sxs/test1
  a exception_handling/pal_sxs/test1/libpaltest_pal_sxs_test1_dll1.dylib
  a exception_handling/pal_sxs/test1/libpaltest_pal_sxs_test1_dll2.dylib
  a exception_handling/pal_sxs/test1/paltest_pal_sxs_test1
  a paltestlist.txt
  a paltests
  a runpaltests.sh
  a runpaltestshelix.sh
/Users/runner/work/1/s/.packages/microsoft.dotnet.helix.sdk/5.0.0-beta.20506.7/tools/Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(18,5): error : You must specify at least one target queue to send a job to helix. Use the HelixTargetQueues property or HelixTargetQueue items. [/Users/runner/work/1/s/src/tests/Common/helixpublishwitharcade.proj]
##[error].packages/microsoft.dotnet.helix.sdk/5.0.0-beta.20506.7/tools/Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(18,5): error : (NETCORE_ENGINEERING_TELEMETRY=Build) You must specify at least one target queue to send a job to helix. Use the HelixTargetQueues property or HelixTargetQueue items.
/Users/runner/work/1/s/.packages/microsoft.dotnet.helix.sdk/5.0.0-beta.20506.7/tools/Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(18,5): error : You must specify at least one target queue to send a job to helix. Use the HelixTargetQueues property or HelixTargetQueue items. [/Users/runner/work/1/s/src/tests/Common/helixpublishwitharcade.proj]
##[error].packages/microsoft.dotnet.helix.sdk/5.0.0-beta.20506.7/tools/Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(18,5): error : (NETCORE_ENGINEERING_TELEMETRY=Build) You must specify at least one target queue to send a job to helix. Use the HelixTargetQueues property or HelixTargetQueue items.
```

@sdmaclea

--- a/eng/pipelines/common/platform-matrix.yml
+++ b/eng/pipelines/common/platform-matrix.yml
@@ -452,7 +452,7 @@ jobs:
 
 # macOS arm64
 
-- ${{ if or(containsValue(parameters.platforms, 'OSX_arm64'), eq(parameters.platformGroup, 'all')) }}:
+- ${{ if containsValue(parameters.platforms, 'OSX_arm64') }}:
   - template: xplat-setup.yml
     parameters:
       jobTemplate: ${{ parameters.jobTemplate }}

